src/marisa_trie.pyx

```diff
@@ -248,33 +254,37 @@ cdef class _Trie:
         self._trie.mmap(c_path)
         return self
 
-    def iterkeys(self, unicode prefix=""):
+    def iterkeys(self, prefix=NoPrefixGiven):
```

Comment:
@liori did you want to make fixes for the tests? I'd be happy to do it if you're low on time just to have this done and out.